src/main/java/seedu/taskboss/logic/parser/AddCommandParser.java

```diff
@@ -33,15 +35,16 @@
     public Command parse(String args) throws InvalidDatesException, DefaultCategoryException {
         ArgumentTokenizer argsTokenizer =
                 new ArgumentTokenizer(PREFIX_PRIORITY, PREFIX_START_DATE,
-                        PREFIX_END_DATE, PREFIX_INFORMATION, PREFIX_CATEGORY);
+                        PREFIX_END_DATE, PREFIX_INFORMATION, PREFIX_RECURRENCE, PREFIX_CATEGORY);
         argsTokenizer.tokenize(args);
         try {
             return new AddCommand(
                     argsTokenizer.getPreamble().get(),
-                    checkEmpty(argsTokenizer.getValue(PREFIX_PRIORITY)),
+                    checkInformationEmpty(argsTokenizer.getValue(PREFIX_PRIORITY)),
```

Comment:
maybe it would be better if you renamed this method. it says check information but your checking priority.